docs/modules/reference/pages/daemons/daemon-config-files/trapd.adoc

```diff
@@ -3,6 +3,19 @@
 = Trapd
 
 Handles processing of xref:operation:events/sources/snmp-traps.adoc[SNMP traps] into events.
+Based around traditional UDP-based SNMP traps, it receives traps that remote devices create.
+It enables OpenNMS to receive SNMP traps and convert them into events, and convert those events into alarms and notifications.
```

Comment:
```suggestion
It enables OpenNMS to receive SNMP traps and convert them into events, and, if configured accordingly, convert those events into alarms and notifications.
```

Maybe something like this? Since alarming and notifying are not configured by default. Not sure about the commas in English :see_no_evil: